libs/designsystem/menu/src/menu.component.html

```diff
@@ -2,18 +2,23 @@
   <ng-content select="button[kirby-button]"></ng-content>
   <ng-container *ngIf="!userProvidedButton">
     <button
+      #defaultButton
+      id="defaultButton"
```

Comment:
We should ensure the id is unique across the page when more then one menu is present:
```suggestion
      [id]="someUniqueIdGeneratedFromUniqueIdGenerator"
```